docs/getting-started.mdx

```diff
@@ -8,46 +8,43 @@ import { AssetsProvider } from '@site/src/components/assets-context';
 import DownloadCard from '@site/src/components/download-card';
 import DownloadCardStyles from '@site/src/css/download-card.module.css';
 
+## Share your network resources within minutes
+
+Easily share private network resources with public internet users, securely, without having to alter any of your local network configuration.
+You can share resource using [NetFoundry's cloud-hosted zrok](https://myzrok.io/),
+or follow our [self-hosted guide](/docs/category/self-hosting/) to manage your own instance.
+
+* Download the binary for your platform [here](https://github.com/openziti/zrok/releases/latest)
+* Sign up for a myzrok.io account [here](https://myzrok.io/) to enable your personal share token
+* Run `zrok enable <your_token>` to enable your shell environment for sharing
+* Run `zrok share public localhost:8080` to start your first share!
+
+![zrok share public](images/zrok_share_public.png)
+
+Access your share from anywhere!
+
+```
+$ curl -s https://dslno640nct4.share.zrok.io | head
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1"/>
+    <meta name="theme-color" content="#000000"/>
+    <meta name="description" content="zrok ui"/>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+```
+
+zrok provides several different share types, including:
+
+* [public shares](./concepts/sharing-public.mdx) for [web services](./concepts/http.md) or [files](./concepts/files.md)
```

Comment:
It may make more sense to the reader for these public/private share links to point at the sections in the same doc that already cover these topics and have links like "more about ____" at the foot of each section that expand further on specific backend modes that are applicable to public and private shares, respectively.

As written, the sections about public and private shares in the same doc are skipped, and these links in the headers point to the separate concept docs that expand on the introductory sections here that were skipped.

Reply:
Still, I've long thought this getting started doc is just too long, so maybe linking the existing sections can be improved by merging the public/private sharing sections with their separate concept docs. This would shorten the getting started doc and then it would make perfect sense for a cluster of links at the top to point to those public/private share concept docs.